users/serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from .models import User
+from posts.serializers import PostSerializer
+
+class PostByAuthorSerializer(serializers.ModelSerializer):
```

Comment:
PostByUser would be a more appropriate name ?

Reply:
Done